plugins/wasd_controls.js

```diff
@@ -3,53 +3,54 @@
 let deletables = [];
 
 let old_animate;
+let rightMouseDown = false;
 
 BBPlugin.register('wasd_controls', {
 	title: 'WASD Controls',
 	icon: 'sports_esports',
 	author: 'JannisX11',
 	description: 'Adds a WASD controlled viewport navigation mode',
 	about: 'The WASD mode can be enabled from the View menu.\nThe keys can be remapped in the keybindings menu.\nThe sensitivity can be changed in the settings under Preview, along with the movement plane.\nHold Control to move faster.',
-	version: '1.1.0',
+	version: '1.2.0',
```

Comment:
Make sure to apply metadata changes like version changes to plugins.json as well.